test-cases/longevity/longevity-alternator-200GB-48h.yaml

```diff
@@ -53,7 +53,7 @@ store_results_in_elasticsearch: false
 use_mgmt: true
 
 server_encrypt: true
-client_encrypt: true
```

Comment:
This is the only place that need to be changed, all other places the change is wrong.

Reply:
Do you want me to change the value in the `param` variable?

Reply:
@Orenef11 on the title of the PR you are saying you gonna change `fix(longevity-alternator-200GB-48h.yaml)`, and in the commit you are changing all the tests cases that uses `client_encrypt`.

which `param` are you asking about ?

Reply:
I need to remove the file name from the commit because I want to disable this flag for all YAML files.
The best solution is to change this value in the `param` variable or there is another way?